### ISSUE TITLE
Export hodlers of a specifc xUDT token on a specific block number

### ISSUE THREAD

**Issue by Keith-CY** [Magickbase/ckb-explorer-public-issues#641]
This feature request comes from the community.

--------

The holder rank at a specific block number is desirable for some activities, e.g airdrop, and the following data are required

1. holder address: CKB address, and its BTC address if it's a RGB++ Address;
2. token amount: raw amount and formatted one if the token info has been detected;

**Comment by Keith-CY** [comment]
@Sven-TBD Please append an ETA of this feature, API can be published prior to the UI

**Comment by Keith-CY** [comment]
Move this issue to high priority

**Comment by Kirl70** [comment]
Design Draft(12-4--12-6):
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=37291%3A48647&mode=design&t=3EagBbH36bdpGb5O-1
<img width="1453" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/106060898/898b57e5-0c2a-4525-a824-0adcd1a7b9b8">

**Comment by rabbitz** [comment]
PR: https://github.com/nervosnetwork/ckb-explorer/pull/1832

#### Export xudt snapshot


```ruby
# api
GET /api/v1/xudts/snapshot?id=&number=

# params
id # xudt type_hash
number # special block number
```

#### e.g.
```bash
curl --location --request GET 'https://ckb-explorer-api-staging.magickbase.com/api/v1/xudts/snapshot?id=0x78e21efcf107e7886eadeadecd1a01cfb88f1e5617f4438685db55b3a540d202&number=13167489' \
--header 'Accept: application/vnd.api+json' \
--header 'User-Agent: Apifox/1.0.0 (https://apifox.com)' \
--header 'Content-Type: application/vnd.api+json' \
--header 'Host: ckb-explorer-api-staging.magickbase.com' \
--header 'Connection: keep-alive'
```

**Comment by FrederLu** [comment]
https://pudge.explorer.nervos.org/xudt/0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351
<img width="1312" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/13d2f2e5-f939-4da7-9f4a-d2c9616b4f00">

https://pudge.explorer.nervos.org/export-xudt-holders?height=1&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351

<img width="928" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/7771806a-3ccf-4ede-8e04-fb7e2a9aa194">

1、This xUDT starts from `block 13,344,852`, but when exporting data, it lacks judgment on exportable data and block data.

---
https://pudge.explorer.nervos.org/zh/export-xudt-holders?height=11&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351

https://pudge.explorer.nervos.org/en/export-xudt-holders?height=11
<img width="903" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/e56b7c95-abd4-43d0-ae2c-b7b4388342cd">

2、Switching the language will cause the URL to change, making it impossible to export data.

---
UI link  https://www.figma.com/design/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=37293-49242&t=spIVoU69fJqsehZ2-0
<img width="320" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/60fdff0b-3bce-4085-8a82-b9c9c7f29276">
3、There is missing information under `Download Data (Holders by Block)` on the mobile version.  At the same time, the style of the red box below in the screenshot does not match the UI, and there is also a lack of dividing lines.
<img width="316" alt="image" src="https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/2eb8c801-ee60-41f5-b619-ba8306ea2931">

**Comment by FrederLu** [comment]
https://pudge.explorer.nervos.org/en/xudt/0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351?page=2
![Image](https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/50885c26-4991-46c9-a8b5-84ce9571cb13)

https://pudge.explorer.nervos.org/en/export-xudt-holders?height=13343194&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351

[exported-xudt-holders-0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351-block-13343194.csv](https://github.com/Magickbase/ckb-explorer-public-issues/files/15429451/exported-xudt-holders-0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351-block-13343194.csv)

![Image](https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/f73ce016-c9c8-47b2-90ea-abd02758f1ec)

4、The exported data is inconsistent with the data on the page.

----
https://pudge.explorer.nervos.org/en/xudt/0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351?page=2
![Image](https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/e8d8b9ed-f828-45b4-8fee-17d80eb4c6bb)

https://pudge.explorer.nervos.org/en/export-xudt-holders?height=13343174&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351
![Image](https://github.com/Magickbase/ckb-explorer-public-issues/assets/25676576/45fef3f4-0661-44f6-b5fe-7ac583df0326)

5、In the exported data, it was found that the `amount` data of the `ckt1qrejnmlar3r452tcg57gvq8patctcgy8acync0hxfnyka35ywafvkqgyfn47vxv2hxxrskahgkvleva8twl46w7mqqdlnesy` address was inconsistent with the page.

**Comment by PainterPuppets** [comment]
> https://pudge.explorer.nervos.org/xudt/0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351 <img alt="image" width="1312" src="https://private-user-images.githubusercontent.com/25676576/333444519-13d2f2e5-f939-4da7-9f4a-d2c9616b4f00.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0NDQ1MTktMTNkMmYyZTUtZjkzOS00ZGE3LTlmNGEtZDJjOTYxNmI0ZjAwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2MTUxN2Q1M2VhNTM4ZmM4ZjI3MzRiYjA3MWIwODhjNGNkZDljMDA1MTk4M2FlOGM2YmY2NzhlZTFiMjhhMjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FOLobcS0bvjVvOrB09lNP7G9iW7vhTzfYbBj-QNPr44">
> 
> https://pudge.explorer.nervos.org/export-xudt-holders?height=1&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351
> 
> <img alt="image" width="928" src="https://private-user-images.githubusercontent.com/25676576/333443314-7771806a-3ccf-4ede-8e04-fb7e2a9aa194.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0NDMzMTQtNzc3MTgwNmEtM2NjZi00ZWRlLThlMDQtZmI3ZTJhOWFhMTk0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4Mjg3ZGFiMjIxMjdkNjhiMGEyN2E0NGVjNTcyYzRjMzg5ZWM1MDgzMTY4OTY3MjZmYTVmMzg1N2IwMGFkMTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RyvdSFgB31Y1l3umsaYhQiCNa-U57tktuVZoHdFJLiM">
> 1、This xUDT starts from `block 13,344,852`, but when exporting data, it lacks judgment on exportable data and block data.
> 
> https://pudge.explorer.nervos.org/zh/export-xudt-holders?height=11&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351
> 
> https://pudge.explorer.nervos.org/en/export-xudt-holders?height=11 <img alt="image" width="903" src="https://private-user-images.githubusercontent.com/25676576/333445036-e56b7c95-abd4-43d0-ae2c-b7b4388342cd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0NDUwMzYtZTU2YjdjOTUtYWJkNC00M2QwLWFlMmMtYjdiNDM4ODM0MmNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwMzlkYzU5NmRhYmVlMjdjNjM5NmE2ZmNkNTI1M2RjNmQ0ODRkM2RiZjQxYTA5NTk2ZTA2MGVhMmM4NGQ3NDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oP_afHWp4D3whSIdkC3t05GlXTMWvbK6l24ratG-p2Y">
> 
> 2、Switching the language will cause the URL to change, making it impossible to export data.
> 
> UI link https://www.figma.com/design/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=37293-49242&t=spIVoU69fJqsehZ2-0 <img alt="image" width="320" src="https://private-user-images.githubusercontent.com/25676576/333437834-60fdff0b-3bce-4085-8a82-b9c9c7f29276.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0Mzc4MzQtNjBmZGZmMGItM2JjZS00MDg1LThhODItYjljOWM3ZjI5Mjc2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1ZWNhNjg1NDU5NjQzZmZlZDIwNDY3NDBlZjExOTI4YWJiNjk3NDI1NTUzZmNjYmFhOTQ5NDA5M2Y2MDRiZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l03d5EiQ01K9u0waNh8RlLuOWyScQol9b8JsnJv4qD4"> 3、There is missing information under `Download Data (Holders by Block)` on the mobile version. At the same time, the style of the red box below in the screenshot does not match the UI, and there is also a lack of dividing lines. <img alt="image" width="316" src="https://private-user-images.githubusercontent.com/25676576/333438230-2eb8c801-ee60-41f5-b619-ba8306ea2931.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0MzgyMzAtMmViOGM4MDEtZWU2MC00MWY1LWI2MTktYmE4MzA2ZWEyOTMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3ODk1N2ZjZjBkYzMyNmIxMmQ1ZmQ5OTYwNWE5NjNjYmEyMWJmNDk1ZThmOGU5ZjYzYjE3YWMxOTlmNWEzZTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iMTEZ0u8uFZ8amfjItJxblbhH2_vQrbYxZxvoCEuuaQ">

fixed by https://github.com/Magickbase/ckb-explorer-frontend/pull/352

**Comment by FrederLu** [comment]
> > https://pudge.explorer.nervos.org/xudt/0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351 <img alt="image" width="1312" src="https://private-user-images.githubusercontent.com/25676576/333444519-13d2f2e5-f939-4da7-9f4a-d2c9616b4f00.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0NDQ1MTktMTNkMmYyZTUtZjkzOS00ZGE3LTlmNGEtZDJjOTYxNmI0ZjAwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY2MTUxN2Q1M2VhNTM4ZmM4ZjI3MzRiYjA3MWIwODhjNGNkZDljMDA1MTk4M2FlOGM2YmY2NzhlZTFiMjhhMjMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FOLobcS0bvjVvOrB09lNP7G9iW7vhTzfYbBj-QNPr44">
> > https://pudge.explorer.nervos.org/export-xudt-holders?height=1&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351
> > <img alt="image" width="928" src="https://private-user-images.githubusercontent.com/25676576/333443314-7771806a-3ccf-4ede-8e04-fb7e2a9aa194.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0NDMzMTQtNzc3MTgwNmEtM2NjZi00ZWRlLThlMDQtZmI3ZTJhOWFhMTk0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4Mjg3ZGFiMjIxMjdkNjhiMGEyN2E0NGVjNTcyYzRjMzg5ZWM1MDgzMTY4OTY3MjZmYTVmMzg1N2IwMGFkMTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RyvdSFgB31Y1l3umsaYhQiCNa-U57tktuVZoHdFJLiM">
> > 1、This xUDT starts from `block 13,344,852`, but when exporting data, it lacks judgment on exportable data and block data.
> > https://pudge.explorer.nervos.org/zh/export-xudt-holders?height=11&id=0xe6396293287fefb9f26d98eb0318fe80890908f0849226ad0c8cab2d62f1e351
> > https://pudge.explorer.nervos.org/en/export-xudt-holders?height=11 <img alt="image" width="903" src="https://private-user-images.githubusercontent.com/25676576/333445036-e56b7c95-abd4-43d0-ae2c-b7b4388342cd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0NDUwMzYtZTU2YjdjOTUtYWJkNC00M2QwLWFlMmMtYjdiNDM4ODM0MmNkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEwMzlkYzU5NmRhYmVlMjdjNjM5NmE2ZmNkNTI1M2RjNmQ0ODRkM2RiZjQxYTA5NTk2ZTA2MGVhMmM4NGQ3NDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oP_afHWp4D3whSIdkC3t05GlXTMWvbK6l24ratG-p2Y">
> > 2、Switching the language will cause the URL to change, making it impossible to export data.
> > UI link https://www.figma.com/design/6XNoimRDbFTTNm016rbIdU/Magickbase?node-id=37293-49242&t=spIVoU69fJqsehZ2-0 <img alt="image" width="320" src="https://private-user-images.githubusercontent.com/25676576/333437834-60fdff0b-3bce-4085-8a82-b9c9c7f29276.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0Mzc4MzQtNjBmZGZmMGItM2JjZS00MDg1LThhODItYjljOWM3ZjI5Mjc2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1ZWNhNjg1NDU5NjQzZmZlZDIwNDY3NDBlZjExOTI4YWJiNjk3NDI1NTUzZmNjYmFhOTQ5NDA5M2Y2MDRiZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.l03d5EiQ01K9u0waNh8RlLuOWyScQol9b8JsnJv4qD4"> 3、There is missing information under `Download Data (Holders by Block)` on the mobile version. At the same time, the style of the red box below in the screenshot does not match the UI, and there is also a lack of dividing lines. <img alt="image" width="316" src="https://private-user-images.githubusercontent.com/25676576/333438230-2eb8c801-ee60-41f5-b619-ba8306ea2931.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1Njk4NjYsIm5iZiI6MTcxNzU2OTU2NiwicGF0aCI6Ii8yNTY3NjU3Ni8zMzM0MzgyMzAtMmViOGM4MDEtZWU2MC00MWY1LWI2MTktYmE4MzA2ZWEyOTMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDA2MzkyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU3ODk1N2ZjZjBkYzMyNmIxMmQ1ZmQ5OTYwNWE5NjNjYmEyMWJmNDk1ZThmOGU5ZjYzYjE3YWMxOTlmNWEzZTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iMTEZ0u8uFZ8amfjItJxblbhH2_vQrbYxZxvoCEuuaQ">
> 
> fixed by [Magickbase/ckb-explorer-frontend#352](https://github.com/Magickbase/ckb-explorer-frontend/pull/352)


It was found that the modified content on this PR was not submitted to the Testnet/Mainnet environment.

please confirm.  @PainterPuppets
